packages/flutter_tools/lib/src/artifacts.dart

```diff
@@ -274,8 +282,13 @@ class CachedArtifacts extends Artifacts {
       case Artifact.linuxDesktopPath:
       case Artifact.windowsDesktopPath:
       case Artifact.flutterMacOSPodspec:
+        // TODO(jonahwilliams): remove once debug desktop artifacts are uploaded
```

Comment:
Issue link?

Reply:
Added link to issue https://github.com/flutter/flutter/issues/38935